_posts/2015-04-14-github-101.markdown

```diff
@@ -0,0 +1,127 @@
+
+Licența folosită determină condițiile în care o altă persoană poate folosi proiectul vostru. Un ghid alegerea unei licențe poate fi găsit aici: [choosealicense.com](http://choosealicense.com/).
+
+{: .center}
+![Crearea unui nou repository](/assets/images/github-101/02-new-repo.png)
+
+
+
+După apăsarea butonului „**Create Repository**” veți ajunge pe pagina repository-ului nou creat.
+
+{: .center}
+![Repository-ul a fost creat](/assets/images/github-101/03-repo-created.png)
+
+
+
+## 3. Instalarea și configurarea GitHub for Windows
+În continuare vom descăra și instala aplicația GitHub for Windows de la adresa [windows.github.com](https://windows.github.com/). După instalare va trebui să vă autentificați în aplicație cu username-ul/adresa de mail și parola alese la crearea contului de GitHub.
+
+{: .center}
+![Autentificare GitHub for Windows](/assets/images/github-101/04-github-setup.png)
+
+
+
+În următorul pas va trebui să configurați identitatea voastră. Scrieți numele întreg și adresa de email. Acestea vor fi publice tuturor persoanelor care au acces la repository.
+
+{: .center}
+![Configurarea GitHub for Windows](/assets/images/github-101/05-github-setup-2.png)
+
+
+
+La ultimul pas puteți apăsa pe „**Skip**”, deoarece încă nu aveți niciun repository local.
+
+{: .center}
+![Importarea repo](/assets/images/github-101/06-github-setup-3.png)
+
+
+
+## 4. Crearea primul commit
+Acum vom clona (crea o copie locală) repository-ul creat anterior. Din aplicație vom da click pe „**+**”, vom selecta tab-ul „**Clone**”, contul și repository-ul pe care dorim să-l clonăm, iar în final vom apăsa „**Clone <repo-name>**”. Pe ecran va apărea o fereastră pentru selectarea directorului unde se va face clonarea.
+
+{: .center}
+![Clonarea unui repository](/assets/images/github-101/07-github-clone.png)
+
+
+
+Acum puteți adăuga fișierele sursă în folderul în care ați clonat repository-ul. În acest exemplu este vorba de fișierul *hello.c*.
+
+{: .center}
+![Adăugarea fișierelor sursă](/assets/images/github-101/08-add-source-files.png)
+
+
+
+Reveniniți în aplicația GitHub for Windows și selectați repository-ul. Veți observa pe coloana centrală că a detectat modificarea unor fișiere („*Uncommitted changes*”). Dați click pe Show pentru a vedea lista lor.
+
+În coloana din dreapta vor apărea toate fișierele noi apărute în directorul repository-ului (ex: *hello.c*), precum și fișierele existente care au fost modificate de la ultimul commit (ex: *README.md*). Pentru a adăuga toate fișierele în repository bifați „*Files to commit*”. Dacă doriți un control mai fin asupra fișierelor puteți expanda fiecare fișier și selecta doar anumite linii. La final scrieți un mesaj de commit în câmpul Summary de pe coloana centrală și dați click pe commit.
+
+{: .center}
+![Crearea unui commit](/assets/images/github-101/09-commit.png)
+
+
+
+Ultimul pas este sincronizarea repository-ului local cu GitHub. Tot ce trebuie făcut este să dați click pe „**Sync**”.
```

Comment:
As introduce notiunea de push sa stie ca este vorba de sync cand vor gasi pe net push. Cred ca si in dictionar ar fi bine sa fie.